pkg/datastore/apikey.go

```diff
@@ -35,6 +36,21 @@ func (a *apiKeyCollection) Factory() Factory {
 	}
 }
 
+func (a *apiKeyCollection) GetStoredFileNames(id string) []string {
+	return []string{id}
+}
+
+func (a *apiKeyCollection) GetUpdatableFileName(id string) (string, error) {
+	switch a.requestedBy {
+	case TestCommander:
```

Comment:
Sorry, I cannot understand this TestCommander why exists. 
If it's just for testing, I think it better that we should write test which cover all commanders.
Hence we don't need to write this case. wdyt?

Reply:
Nice point, lets me confirm one thing, your suggestion is to treat this `TestCommander` as a case in default, and in case we want to test it, should we make tests with all possible commanders, is that right? In that case, I totally agree with you 👍

Reply:
Almost yes. imo `TestCommander` is not be required. 
For instance, I think we can write it as below. And we make the test all cases. 
```
switch cmd {
        // Cover all cases in commander not to overlook any cases.
	case WebCommander, PipectlCommander:
		return fmt.Sprintf("%s_api", id), nil
	case OpsCommander:
		return fmt.Sprintf("%s_ops", id), nil
	case PipedCommander:
		return fmt.Sprintf("%s_piped", id), nil
}
return "", ErrUnsupported
```

Reply:
For sure 👍

Reply:
Updated by https://github.com/pipe-cd/pipecd/pull/3272/commits/f45776f1d14e5065873bf51d6d77dcdaaef2a2ef 🙏